addons/binding/org.openhab.binding.mysensors/src/main/java/org/openhab/binding/mysensors/MySensorsBindingConstants.java

```diff
@@ -0,0 +1,384 @@
+    public static final String CHANNEL_WIND = "wind";
+    public static final String CHANNEL_GUST = "gust";
+    public static final String CHANNEL_RAIN = "rain";
+    public static final String CHANNEL_RAINRATE = "rainrate";
+    public static final String CHANNEL_UV = "uv";
+    public static final String CHANNEL_WEIGHT = "weight";
+    public static final String CHANNEL_IMPEDANCE = "impedance";
+    public static final String CHANNEL_CURRENT = "current";
+    public static final String CHANNEL_DISTANCE = "distance";
+    public static final String CHANNEL_LIGHT_LEVEL = "light-level";
+    public static final String CHANNEL_VERSION = "version";
+    public static final String CHANNEL_BATTERY = "battery";
+    public static final String CHANNEL_HVAC_FLOW_STATE = "hvac-flow-state";
+    public static final String CHANNEL_HVAC_FLOW_MODE = "hvac-flow-mode";
+    public static final String CHANNEL_HVAC_SETPOINT_HEAT = "hvac-setPoint-heat";
+    public static final String CHANNEL_HVAC_SETPOINT_COOL = "hvac-setPoint-cool";
+    public static final String CHANNEL_HVAC_SPEED = "hvac-speed";
+    public static final String CHANNEL_VAR1 = "var1";
+    public static final String CHANNEL_VAR2 = "var2";
+    public static final String CHANNEL_VAR3 = "var3";
+    public static final String CHANNEL_VAR4 = "var4";
+    public static final String CHANNEL_VAR5 = "var5";
+    public static final String CHANNEL_FLOW = "flow";
+    public static final String CHANNEL_VOLUME = "volume";
+    public static final String CHANNEL_LOCK_STATUS = "lock-status";
+    public static final String CHANNEL_LEVEL = "level";
+    public static final String CHANNEL_RGB = "rgb";
+    public static final String CHANNEL_RGBW = "rgbw";
+    public static final String CHANNEL_ID = "id";
+    public static final String CHANNEL_UNIT_PREFIX = "unit-prefix";
+    public static final String CHANNEL_TEXT = "text";
+    public static final String CHANNEL_CUSTOM = "custom";
+    public static final String CHANNEL_POSITION = "position";
+    public static final String CHANNEL_IR_RECORD = "ir-record";
+    public static final String CHANNEL_PH = "ph";
+    public static final String CHANNEL_ORP = "orp";
+    public static final String CHANNEL_EC = "ec";
+    public static final String CHANNEL_VAR = "var";
+    public static final String CHANNEL_VA = "va";
+    public static final String CHANNEL_POWER_FACTOR = "power-factor";
+    public static final String CHANNEL_IR_SEND = "irSend";
+    public static final String CHANNEL_IR_RECEIVE = "irReceive";
+
+    // Extra channel names for non-standard MySensors channels
+    public static final String CHANNEL_MYSENSORS_MESSAGE = "mySensorsMessage";
+    public static final String CHANNEL_LAST_UPDATE = "lastupdate";
+
+    /**
+     * Mapping MySensors message type/subtypes to channels.
+     */
+    public static final Map<MySensorsMessageSubType, String> CHANNEL_MAP = new HashMap<MySensorsMessageSubType, String>() {

```

Comment:
Nothing, I do not use anonymous inner classes for such initialisations so I was not yet aware that you could not use with the diamond operator if eclipse or the Jdk says so, I believe him.